docs/apache-airflow/concepts/pools.rst

```diff
@@ -33,14 +37,16 @@ Tasks can then be associated with one of the existing pools by using the ``pool`
     )
     aggregate_db_message_job.set_upstream(wait_for_empty_queue)
 
-The ``pool`` parameter can be used in conjunction with the ``priority_weight`` parameter to define priorities in the queue, and which tasks get executed first as slots open up in the pool.
-
-The default ``priority_weight`` is ``1``, and can be bumped to any number. When sorting the queue to evaluate which task should be executed next, we use the ``priority_weight``, summed up with all of the ``priority_weight`` values from tasks downstream from this task; the highest summed value wins. Thus, you can bump a specific important task, and the whole path to that task gets prioritized accordingly.
 
-Tasks will be scheduled as usual while the slots fill up. Once capacity is reached, runnable tasks get queued and their state will show as such in the UI. As slots free up, queued tasks start running based on the ``priority_weight`` (of the task and its descendants).
+Tasks will be scheduled as usual while the slots fill up. Once capacity is reached, runnable tasks get queued and their
+state will show as such in the UI. As slots free up, queued tasks start running based on the
+:ref:`concepts:priority-weight`.
+(of the task and its descendants).
```

Comment:
I think it's worth mentioning that the number of slots occupied by a task can be configured by `pool_slots`

Reply:
I didn't know about this feature! Will add to the doc.